nvssclient/Services/TimedHostedService.cs

```diff
@@ -183,13 +189,19 @@ public async void ResendMessages()
                         context.Update(item);
                         context.SaveChanges();
                     }
-                    else if (response.StatusCode == HttpStatusCode.Unauthorized)
-                    {
-                        _logger.LogError($">>> Unauthorized error submitting {message.MessageId}, status: {response.StatusCode}");
-                    }
                     else
                     {
-                        _logger.LogError($">>> Error submitting {message.MessageId}, status: {response.StatusCode}");
+                        if (response.StatusCode == HttpStatusCode.Unauthorized)
+                        {
+                            _logger.LogError($">>> Unauthorized error submitting {message.MessageId}, status: {response.StatusCode}");
+                        }
+                        else
```

Comment:
same as above, only set to error if the status code was 400, other errors should just be logged so the message can be resent

Reply:
Should be resolved